ports-of-call/portable_arrays.hpp

```diff
@@ -262,20 +208,78 @@ class PortableMDArray {
                             const int indx, const int nvar);
 
  private:
+  template <typename... NX, std::size_t N = sizeof...(NX)>
+  PORTABLE_FORCEINLINE_FUNCTION auto make_nxs_array(NX... nxs) {
+    std::array<std::size_t, MAXDIM> a;
+    std::array<std::size_t, N> t{static_cast<std::size_t>(nxs)...};
+    for (auto i = 0; i < N; ++i) {
```

Comment:
I'm gonna insist this be `int` or `size_t`. IMO this is not a style thing but a self-documenting thing. It's an index it's not a double.